### PR TITLE
[AMORO-2980] Support fuzzy filtering db or table in the Optimizing->Tables page

### LINKED ISSUE #2980: [Subtask]: Support fuzzy filtering db or table in the Optimizing->Tables page
### Search before asking

- [X] I have searched in the [issues](https://github.com/apache/amoro/issues?q=is%3Aissue) and found no similar issues.

### Description

As titled

### Parent issue

https://github.com/apache/amoro/issues/885

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by tcodehuber**
## Why are the changes needed?

Close #2980.

## Brief change log

-

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by tcodehuber** [comment]
image

**Review by xleoken — APPROVED** [review]

File: amoro-ams/amoro-ams-dashboard/src/services/optimize.service.ts [hunk 1]
```diff
@@ -25,12 +25,14 @@ export function getOptimizerGroups() {
...
+    dbSearchInput: string
+    tableSearchInput: string
     page: number
     pageSize: number
   },
 ) {
-  const { optimizerGroup, page, pageSize } = params
-  return request.get(`ams/v1/optimize/optimizerGroups/${optimizerGroup}/tables`, { params: { page, pageSize } })
+  const { optimizerGroup, dbSearchInput, tableSearchInput, page, pageSize } = params
+  return request.get(`ams/v1/optimize/optimizerGroups/${optimizerGroup}/tables`, { params: {dbSearchInput, tableSearchInput, page, pageSize } })
```
Comment (zhangmo8) [inline amoro-ams/amoro-ams-dashboard/src/services/optimize.service.ts:35]: Remember to format the code before submitting

**Review by zhangmo8 — APPROVED** [review]
LGTM

File: amoro-ams/amoro-ams-dashboard/src/services/optimize.service.ts (hunk 1 above)
Comment (zhangmo8) [reply amoro-ams/amoro-ams-dashboard/src/services/optimize.service.ts:35]: `pnpm run format`

**Pull request merged**